src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -117,16 +117,18 @@ public void start() throws Exception {
         ensureExchange(dlx);
 
         this.publishChannel = connection.newChannel();
-        connection.ensure(queueName + "_SIDELINE", queueName, dlx,
-                connection.rmqOpts(config));
+        String sidelineQueueName = NamingUtils.getSideline(queueName);
         if (config.isSharded()) {
             int bound = config.getShardCount();
+            connection.ensure(sidelineQueueName, queueName, dlx, connection.rmqOpts(config));
```

Comment:
Same line is present in if else block. Extract it out

Reply:
Done.